dagshub/pytorch_lightning/logger.py

```diff
@@ -35,10 +38,28 @@ def __init__(self,
         self.logger = LoggerImpl(metrics_path=metrics_path, should_log_metrics=should_log_metrics,
                                  hparams_path=hparams_path, should_log_hparams=should_log_hparams,
                                  should_make_dirs=should_make_dirs, eager_logging=False)
+        self._name = name or ""
+        self._version = version
+        self._experiment = None
```

Comment:
I guess this isn't necessary if we are going to always return the `self.logger` in the `experiment` function.